client/src/features/sessionsV2/PauseOrDeleteSessionModal.tsx

```diff
@@ -281,12 +273,13 @@ function PauseSessionModalContent({
           session (new and edited files) will be preserved while the session is
           paused.
         </p>
-        {hibernatedSecondsThreshold > 0 && (
-          <InfoAlert dismissible={false} timeout={0}>
-            Please note that paused session are deleted after{" "}
-            {hibernationThreshold} of inactivity.
-          </InfoAlert>
-        )}
+        {session?.status?.will_hibernate_at &&
+          session?.status?.will_hibernate_at?.length > 0 && (
```

Comment:
No need for a length check on strings:

```suggestion
        {session?.status?.will_hibernate_at && (
```